aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -177,7 +177,11 @@ Tensor& set_storage_quantized_(
   auto* self_ = self.unsafeGetTensorImpl();
   self_->set_storage_keep_dtype(std::move(storage));
   self_->set_storage_offset(storage_offset);
-  self_->set_sizes_and_strides(sizes, strides);
+  if(strides.data() == nullptr){
+    self_->set_sizes_contiguous(sizes);
+  }else{
```

Comment:
best you align the coding style with other code.
```suggestion
  } else {
```

Reply:
great thanks for the suggestion, has modified